src/status_im/components/toolbar_new/view.cljs

```diff
@@ -94,10 +94,11 @@
              [icon-action icon icon-opts handler])
        {:key (str "action-" (or image icon))}))])
 
-(defn toolbar2
-  ([title] (toolbar2 nil title))
-  ([props title] (toolbar2 props default-nav-back [content-title title]))
-  ([props nav-item content-item] (toolbar2 props nav-item content-item [actions [{:image :blank}]]))
+(defn toolbar
+  ([] (toolbar nil))
+  ([title] (toolbar nil title))
+  ([props title] (toolbar props default-nav-back [content-title title]))
```

Comment:
I know it's a WIP and this is current interface, but the change of order throws me off. Why is it title first arg, then second arg, then not there at all? Seems weird to me.

Reply:
Properties first is a convention we have for most components. First three arities are for convenience.

An alternative would be to have a dedicated method for `title` based toolbar.

Reply:
If properties is more important then we shouldn't allow title as a first argument imo, break expectation. Positional arguments shouldn't change order, way too much mental overhead

Reply:
Alternatively, if we want to have lots of options, turn it into a map so order is irrelevant.

Reply:
Probably we could use two arities here:
`[props]`
`[props nav-item content-item]`
As an option we can define two different components, where the first one will have these first three arities (which looks fine imo), and another one for the last case. Because one function with all these arities inside looks off for sure.